common-items.lic

```diff
@@ -80,13 +80,14 @@ module DRCI
     result =~ /You tap/
   end
 
-  def refill_lockpick_container(lockpick_type, hometown, container)
+  def refill_lockpick_container(lockpick_type, hometown, container, cost = 125)
     app_result = DRC.bput("app my lockpick #{container}", 'and it appears to be full', 'and it might hold an additional \d+', '\d+ lockpicks would probably fit')
     waitrt?
     return unless app_result =~ /(\d+)/
 
     room = get_data('town')[hometown]['locksmithing']['id']
     count = Regexp.last_match(1).to_i
+    ensure_copper_on_hand(cost * count, hometown)
```

Comment:
Rather than introducing a new dependency, you could split `refill_lockpick_container` into two methods, one to count your ring and one to refill.